### PR TITLE
Flame thrower plasma tanks can be removed easily.

### LINKED ISSUE #28385: You can't remove flamethrower plasma tanks
The only way you can change the tank seems to be by deconstructing the flamethrower.

Either that or the way is so convoluted that nobody knows how to do it even after asking on 4 different occasions.

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
tweak: Flame thrower plasma tanks can be removed with alt-click.
/:cl:

fixes #28385

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]
Swap the functionality so as not to bamboozle old players

File: code/game/objects/items/weapons/flamethrower.dm [hunk 1]
```diff
@@ -137,6 +137,16 @@
 /obj/item/weapon/flamethrower/attack_self(mob/user)
 	toggle_igniter(user)
 
+/obj/item/weapon/flamethrower/AltClick(mob/user)
+	if(ptank)
```
Comment (AnturK) [inline code/game/objects/items/weapons/flamethrower.dm]: Needs ghost check and adjacency check.

**Comment by Cyberboss** [comment]
@Expletive

File: code/game/objects/items/weapons/flamethrower.dm (hunk 1 above)
Comment (Expletive) [inline code/game/objects/items/weapons/flamethrower.dm]: How do I check if someone's a ghost or not adjacent?

Comment (AnturK) [inline code/game/objects/items/weapons/flamethrower.dm]: Check type for living, user.incapacitated() and Adjacent(user).

**Pull request merged**